### PR TITLE
Updates of the wiki files in English (auth)

### PR DESCRIPTION
**Pull request by Tony931023**
### All updates of the wiki files in English (auth)

- [x] account.md https://github.com/azerothcore/wiki/pull/571/commits/af85448673d25ef90aa83652b4cec5d12f29f2df
- [x] account_access.md https://github.com/azerothcore/wiki/pull/573/commits/ab7d0d56fe12a7ae96ea93c9b60fb90df221046a
- [x] account_banned.md (no changes required)
- [x] account_data.md (no changes required)
- [x] autobroadcast.md (no changes required)
- [x] build_info.md (no changes required)
- [x] ip_banned.md https://github.com/azerothcore/wiki/pull/573/commits/cfe0a3c678bb3798bf4a154697bb5a2cd464cf54
- [x] logs.md https://github.com/azerothcore/wiki/pull/573/commits/342e9b49b5488562acd31c5164753aca59ae9dd7
- [x] logs_ip_actions.md https://github.com/azerothcore/wiki/pull/573/commits/727cb2cea13c6cf6a6c6847166c9ddc8df9a561c
- [x] realmcharacters.md https://github.com/azerothcore/wiki/pull/573/commits/dd25688f1ab6347b27ee702bd9c4d02b0d695b49
- [x] realmlist.md https://github.com/azerothcore/wiki/pull/573/commits/d21ae5d28e165367f4164e60d2c65be3ba07c34f
- [x] uptime.md https://github.com/azerothcore/wiki/pull/573/commits/22b51ab987c53f189fb3615e822ac1e341dec7c6
- [x] version_db_auth.md https://github.com/azerothcore/wiki/pull/573/commits/1232a33028f08a15af5849b2725d20959ad36cc5
- [x] secret_digest.md  https://github.com/azerothcore/wiki/pull/573/commits/36716f4f049c7453e312e9bd9541bfa5de9d0642 (added)

### Update database-auth.md and create secret_digest.md https://github.com/azerothcore/wiki/pull/573/commits/36716f4f049c7453e312e9bd9541bfa5de9d0642

### CODE REVIEW

File: docs/account_access.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This table holds security access level for any realm in [realmlist](realmlist) t
 | [id][1]      | INT          | UNSIGNED   | PRI | NO   |         |       |         |
 | [gmlevel][2] | TINYINT      | UNSIGNED   |     | NO   |         |       |         |
 | [RealmID][3] | INT          | SIGNED     | PRI | NO   | -1      |       |         |
-| [comment][4] | VARCHAR(255) |            |     | YES  |         |       |         |
+| [comment][4] | VARCHAR(255) |            |     | YES  | ""      |       |         |
```
Comment (Kitzunu) [inline docs/account_access.md:16]: ''

File: docs/version_db_auth.md [hunk 2]
```diff
@@ -8,21 +8,34 @@
...
-### 2020_02_07_00
+### date 
+
+sql revisions date
+
+### 2021_06_17_00
+
+the date of the last update to the auth database
 
 ### Example
```
Comment (Kitzunu) [inline docs/version_db_auth.md:35]: Remove this, unnecessary and doesn't serve any point.

File: docs/version_db_auth.md [hunk 3]
```diff
@@ -8,21 +8,34 @@
...
 ### required_rev
 
-### 2020_02_07_00
+### date 
+
+sql revisions date
+
+### 2021_06_17_00
+
+the date of the last update to the auth database
```
Comment (Kitzunu) [inline docs/version_db_auth.md:33]: Grammar... We start a sentence with an uppercase letter and end it with a dot.

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: docs/version_db_auth.md (hunk 2 above)
Comment (Kitzunu) [reply docs/version_db_auth.md:35]: not this lol

File: docs/version_db_auth.md [hunk 4]
```diff
@@ -8,21 +8,34 @@
...
 
-| sql_rev             | required_rev | 2020_02_07_00 |
-|---------------------|--------------|---------------|
-| 1554142988374631100 |              |               |
-| 1579213352894781043 |              |               |
+| sql_rev             | required_rev | date          | 2021_06_17_00 |
+|---------------------|--------------|---------------|---------------|
+| 1554142988374631100 |              |               |               |
+| 1579213352894781043 |              |               |               |
+| 1620146306002634000 |              | 2021_06_17_00 |               |
```
Comment (Kitzunu) [inline docs/version_db_auth.md:41]: Remove this, unnecessary and doesn't serve any point.

File: docs/version_db_auth.md [hunk 5]
```diff
@@ -20,22 +20,17 @@
...
 
-sql review
+Sql review.
 
 ### required_rev
 
 ### date 
 
-sql revisions date
+Sql revisions date.
```
Comment (Kitzunu) [inline docs/version_db_auth.md:29]: SQL

File: docs/version_db_auth.md (hunk 4 above)
Comment (Tony931023) [reply docs/version_db_auth.md]: fix https://github.com/azerothcore/wiki/pull/573/commits/fed2f3ec6198906fc7275f4d0f5bcd556a354d23

File: docs/version_db_auth.md (hunk 5 above)
Comment (Tony931023) [reply docs/version_db_auth.md:29]: fix https://github.com/azerothcore/wiki/pull/573/commits/1047552e9abc899648102bd35a2758b8009d7a91

File: docs/version_db_auth.md [hunk 6]
```diff
@@ -8,21 +8,29 @@
...
 | [sql_rev][1]       | VARCHAR(100) |            | PRI | NO   |         |       |         |
 | [required_rev][2]  | VARCHAR(100) |            | MUL | YES  |         |       |         |
-| [2020_02_07_00][3] | bit(1)       |            |     | YES  |         |       |         |
+| [date][3]          | VARCHAR(50)  |            |     | YES  |         |       |         |
+| [2021_06_17_00][4] | BIT(1)       |            |     | YES  |         |       |         |
 
 [1]: #sql_rev
 [2]: #required_rev
 [3]: #2020_02_07_00
+[4]: #2021_06_17_00
```
Comment (Kitzunu) [inline docs/version_db_auth.md]:
```suggestion
[1]: #sqlrev
[2]: #requiredrev
[3]: #date
[4]: #2021061700
```

Comment (Kitzunu) [reply docs/version_db_auth.md]: _ doesnt work in links, so you just cut them from the link and it will work :D

Comment (Tony931023) [reply docs/version_db_auth.md]: it didn't work like that https://github.com/azerothcore/wiki/pull/573/commits/a036b578f40be0e76678d59ae01afdf45991fd13 

so if it worked for me https://github.com/azerothcore/wiki/pull/573/commits/4544705eed454997bc0c19728ed371dbd0a97af2

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**